multinet/api.py

```diff
@@ -24,11 +27,62 @@
     AlreadyExists,
     RequiredParamsMissing,
 )
-from multinet.user import current_user
+from multinet.user import current_user, find_user_from_id
 
 bp = Blueprint("multinet", __name__)
 
 
+# Included here due to circular imports
+# TODO: Remove once implementing new ORM and permission storage
+def _permissions_id_to_user(permissons: WorkspacePermissions) -> Dict:
+    """Transform permission documents to directly container user info."""
+
+    new_permissions = deepcopy(permissons)
+    for role, users in new_permissions.items():
+        if role == "public":
+            continue
+
+        if role == "owner":
+            assert isinstance(users, str)
```

Comment:
We shouldn't have any `assert` statements in the main code. What role is it serving here?

Reply:
Ah I think I might understand. Is this something `mypy` makes use of to narrow from Union types?

Reply:
Yeah, it's to deal with `mypy`. Since all other fields besides `public` are arrays, we have to tell `mypy` that in this case, it's just a string, since `mypy` is too stupid to figure that out on its own.